### PR TITLE
jpm/jpmsru.cpp: Added Lite a Line, cleaned up the driver

### PR DESCRIPTION
**Pull request by SomeRandomGuyIdk**
- Added support for Lite a Line, a Dutch game together with a very basic handmade layout
- Some cleanups to the driver
- Added a postload callback to fix j_ewnc not updating reels after loading a save state
- Order of display digit outputs has changed to reflect the actual order in the memory map

### CODE REVIEW

File: src/mame/layout/j_lal.lay [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+<mamelayout version="2">
+	<element name="matrixlamp">
+		<rect state ="0">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.7" green="0.7" blue="0.7" />
+		</rect>
+		<rect state ="1">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.95" green="0.95" blue="0.43" />
+		</rect>
```
Comment (cuavas) [inline src/mame/layout/j_lal.lay:15]: I’m not sure what you’re trying to do with the bounds here – the internal coordinate system of the element will adjust to the bounds of all components.  If you’ve only got `rect` components that fill the element, you may as well omit the bounds and take the default unit square.

File: src/mame/layout/j_lal.lay [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+	<element name="rect_black">
+		<rect>
+			<color red="0.0" green="0.0" blue="0.0"/>
+		</rect>
+	</element>
+	<element name="text_credits">
+		<text string="Credits">
+			<color red="1.0" green="1.0" blue="1.0"/>
+			<bounds x="0.05" y="0.05" width="0.90" height="0.90"/>
+		</text>
```
Comment (cuavas) [inline src/mame/layout/j_lal.lay:31]: Same for these text elements – if you want to inset the text from the bounds of the element, you need to add a transparent rectangle to force the bounds (e.g. <rect><color alpha="0" /></rect>`).  The way you have it now, the elements bounds will be (0.05 0.05 0.95 0.95) so there will be no space around the text anyway.

File: src/mame/layout/j_lal.lay (hunk 1 above)
Comment (SomeRandomGuyIdk) [reply src/mame/layout/j_lal.lay]: Removed these. They came from whichever layout I copy-pasted all of mine from, wasn't paying much attention to these things.

File: src/mame/layout/j_lal.lay (hunk 2 above)
Comment (SomeRandomGuyIdk) [reply src/mame/layout/j_lal.lay]: Removed them, copypaste as well.

**Pull request merged**